packages/components/src/internal/renderers/FileColumnRenderer.tsx

```diff
@@ -81,5 +81,10 @@ export function getAttachmentTitleFromName(name: string): string {
     if (name.indexOf('/') > -1) {
         return name.substr(name.lastIndexOf('/') + 1);
     }
+    // Issue 43725: Windows file name comes through with backslash instead
+    else if (name.indexOf('\\') > -1) {
```

Comment:
TC run shows a single backslash, should this work for a single slash case?
https://teamcity.labkey.org/repository/download/LabKey_Trunk_Premium_SampleManager_SampleManagerCSqlserver/1506964:id/SMSampleUpdateTest.tar.gz!/202108130529SMSampleUpdateTest%23testUpdateSampleFileField.html

Reply:
Yes, I needed to have the double backslash because a single backslash is the escape character. I believe TeamCity is passing now with this change.

Reply:
that makes sense to escape. I didn't see the TC run on sql server, I believe you need to manually trigger it.